test/EFCore.PG.FunctionalTests/Query/SpatialQueryNpgsqlFixture.cs

```diff
@@ -20,19 +20,13 @@ protected override IServiceCollection AddServices(IServiceCollection serviceColl
                 .AddEntityFrameworkNpgsqlNetTopologySuite();
         }
 
-        public override DbContextOptionsBuilder AddOptions(DbContextOptionsBuilder builder)
-        {
-            var optionsBuilder = base.AddOptions(builder);
-            new NpgsqlDbContextOptionsBuilder(optionsBuilder).UseNetTopologySuite();
-
-            return optionsBuilder;
-        }
-
         protected override void OnModelCreating(ModelBuilder modelBuilder, DbContext context)
         {
             base.OnModelCreating(modelBuilder, context);
 
             modelBuilder.HasPostgresExtension("postgis");
         }
+
+        //protected override bool CanExecuteQueryString => true;
```

Comment:
Why?

Reply:
Because we haven't implemented this yet, see #1232